runtime/riscv.S

```diff
@@ -1099,6 +1104,7 @@ END_FUNCTION(caml_resume)
 /* Run a function on a new stack, then either
    return the value or invoke exception handler */
 FUNCTION(caml_runstack)
+        CFI_SIGNAL_FRAME
```

Comment:
Same as arm64: I'd rather do this after the TSAN block, which does not switch stacks yet.